template-manager-cmtemplate/src/test/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/tez/TezRoleConfigProviderTest.java

```diff
@@ -1,12 +1,16 @@
 package com.sequenceiq.cloudbreak.cmtemplate.configproviders.tez;
 
+import static com.sequenceiq.cloudbreak.cmtemplate.configproviders.core.CoreConfigProvider.CORE_DEFAULTFS;
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
```

Comment:
Minor: I suggest replacing standard JUnit assertions with AssertJ's `Assertions.assertThat()` because the latter offers a fluent API and more easily readable. There are several examples in this module, e.g. `HiveKnoxConfigProviderTest`.

Reply:
Addressed!